docs/pages/_document.tsx

```diff
@@ -1,20 +1,19 @@
 import { Global } from '@emotion/core';
-import Document, { Html, Head, Main, NextScript } from 'next/document';
 import { extractCritical } from 'emotion-server';
-
+import Document, { Html, Head, Main, NextScript, DocumentContext } from 'next/document';
 import * as React from 'react';
-import * as Analytics from '~/common/analytics';
 
-import { globalReset } from '~/global-styles/reset';
-import { globalNProgress } from '~/global-styles/nprogress';
-import { globalTables } from '~/global-styles/tables';
+import * as Analytics from '~/common/analytics';
+import { globalExtras } from '~/global-styles/extras';
 import { globalFonts } from '~/global-styles/fonts';
+import { globalNProgress } from '~/global-styles/nprogress';
 import { globalPrism } from '~/global-styles/prism';
+import { globalReset } from '~/global-styles/reset';
+import { globalTables } from '~/global-styles/tables';
 import { globalTippy } from '~/global-styles/tippy';
-import { globalExtras } from '~/global-styles/extras';
 
-export default class MyDocument extends Document {
-  static async getInitialProps(ctx) {
+export default class MyDocument extends Document<{ css?: string }> {
```

Comment:
I don't think we need the `.css` prop anymore, since Emotion 10, right?

Reply:
Right, leftover from pre-emotion10 times 😄